src/content/self-hosted/administration/configuration.mdx

```diff
@@ -47,6 +47,26 @@ subdomain: "example.com"
 
 After installation, we recommend that you create a DNS entry for `*.$SUBDOMAIN`, pointing to the public address of your load balancer.
 
+### publicOverride
```

Comment:
@omnomagonz @pchico83 since this is something we've never officially supported before, could we take the opportunity to give it a more standard and self-descriptive name to the property?  (eg publicURL).